WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadNotifier.java

```diff
@@ -136,19 +163,16 @@ void addMediaInfoToForegroundNotification(@NonNull MediaModel media) {
         startOrUpdateForegroundNotification(null);
     }
 
-    void updateNotificationIcon(PostModel post, Bitmap icon) {
-        // TODO MEDIA reimplement or remove completely
-
-//        NotificationData notificationData = sPostIdToNotificationData.get(post.getId());
-//
-//        if (icon != null) {
-//            notificationData.latestIcon = icon;
-//            mNotificationBuilder.setLargeIcon(notificationData.latestIcon);
-//        }
-//        doNotify(sPostIdToNotificationData.get(post.getId()).notificationId, mNotificationBuilder.build());
-    }
-
-    void incrementUploadedPostCountFromForegroundNotification(@NonNull PostModel post) {
+    void incrementUploadedPostCountFromForegroundNotificationOrFinish(@NonNull PostModel post) {
```

Comment:
I'm confused by `orFinish` being part of this method name. Any chance we can get rid of it, especially since the method name is so long?

Reply:
Sure thing! got it back in 6aa5620 and also renamed the Media method counterpart to keep it the same way